### ISSUE TITLE
[Feature Request] A version of the script that would replace the original name with the english one 

### ISSUE THREAD

**Issue by SacredZenpie** [Animorphs/MAL-English-Titles#12]
A version of the script, that instead of writing above the original title, would replace the original title. Mostly would be useful for the anime relation section (sequels etc) and potentially homepage if it were to be implemented.

**Comment by Animorphs** [comment]
I'm not sure if I'll implement this, but your request has been noted and will be thought about.

**Comment by saitamaoo7** [comment]
yeah it might also fix the issue of translations not showing in places such as anime and manga review page

**Comment by Animorphs** [comment]
@saitamaoo7 assuming you're talking about https://myanimelist.net/reviews.php, that was just an oversight from me. I've just added it in, so thanks for the report.
might still work on the option for full replacement as requested in the original request, but unsure. PR's welcome.

**Comment by Scotty-Trees** [comment]
Just wanted to add that I also would love to see the Japanese name replaced with the English name. It just makes it far easier to read and find what I need. I still watch all anime in subtitles, but just for viewing the site MyAnimeList, I would much prefer to have only the English names shown.
